tools/topology/topology1/sof/pipe-drc-eq-volume-demux-playback.m4

```diff
@@ -44,17 +44,28 @@ C_CONTROLBYTES(MY_MULTIBAND_DRC_CTRL, PIPELINE_ID,
 	,
 	MULTIBAND_DRC_priv)
 
+# DRC Enable switch
+define(MY_MULTIBAND_DRC_ENABLE, concat(`multiband_drc_enable_', PIPELINE_ID))
+C_CONTROLMIXER(MY_MULTIBAND_DRC_ENABLE, PIPELINE_ID,
```

Comment:
is mixer the right type?

Reply:
I learned this from [tdfb_controls.m4](https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/m4/tdfb_controls.m4)